woocommerce/woocommerce
# Fix edit custom field modal width

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes a visual issue because the modal width when editing an existing custom field was too wide.

**Before**

<img width="1587" alt="image-3" src="https://github.com/woocommerce/woocommerce/assets/1314156/c7f58a42-25fc-4cb8-9257-2534b2081670">

**After**

![Screenshot 2024-04-19 at 9 22 21 AM](https://github.com/woocommerce/woocommerce/assets/1314156/52aebae0-0cc4-4150-8f50-da3304555d60)

Closes #46614.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the New product and the custom fields (`product-custom-fields`) are enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Go to Product > Add New > Organization.
3. Create a custom field.
4. Press `Edit` and verify that the modal width looks correct (the minimum width should be 350px).



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
